### ISSUE TITLE
Rename project after `tuist init`

### ISSUE THREAD

**Issue by Paul-van-Klaveren** [tuist/tuist#6016]
### TL;DR 
What is the recommended way to _rename_ an existing tuist project?
Can tuist rename a project, if needed?

### Background
I've ran `tuist init --name WIP`[^1] while the official project name wasn't final yet. Now after some time, I'd like to rename it, but not by deleting everything I've done so far followed by running `tuist init --name Final`. 
`tuist init` used  the provided `WIP` name in various places: the project directory name, inside the `Project.swift`, the main app, and test file. I could manually change these occurrences and rename them, but maybe there is an easier/quicker/less-error-prone way. I've checked the tuis documentation and command line help, but couldn't find it so far.

[^1]: I provided a name, because the command was executed in a differently named repository directory that follows an elobarate company naming scheme.

### Potential solution

Ideally, tuist has a command to rename a project, e.g. `tuist rename <project-name>` or `tuist edit --name <project-name>`.

Meanwhile, at the project description level I could define a constant String with the app name, and  then use that in the parameters of the `Project` definition, but the aforementioned file names would still require manual renaming.

### macOS version

14.3.1

### Tuist version

4.4.0

### Xcode version

15.2

**Comment by pepicrft** [comment]
Thanks for opening this issue @Paul-van-Klaveren. I recommend manually renaming those things across the system directories and Tuist manifest files. I don't think Tuist should provide a command for that because the shape of your project is unknown to Tuist, so we'd never be able to do it reliably, leaving you and any other user with bad taste.
